recipes/virulencefinder/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
You should add `set -euo pipefail` (http://redsymbol.net/articles/unofficial-bash-strict-mode/)